src/stylesheets/components/_accordions.scss

```diff
@@ -19,24 +15,24 @@ $v-padding: $spacing-md-small;
 // scss-lint:disable PropertyCount
 @mixin accordion-button-styles {
   @include button-unstyled;
-  background-color: $color-gray-lightest;
+  background-color: color('gray-5');
   background-image: url('#{$theme-image-path}/minus.png');
   background-image: url('#{$theme-image-path}/minus.svg');
-  background-position: right $h-padding center;
+  background-position: right units(2.5) center;
   background-repeat: no-repeat;
-  background-size: $icon-size;
-  color: $color-base;
+  background-size: units(2);
+  color: $color-base; // theme-base or something?
   cursor: pointer;
   display: inline-block;
-  font-family: $font-sans;
-  font-weight: $font-bold;
+  font-family: $theme-font-sans;
+  font-weight: $theme-font-weight-bold;
   margin: 0;
-  padding: $v-padding ($h-padding * 2 + $icon-size) $v-padding $h-padding;
+  padding: units(2) (units(2.5) * 2 + units(2)) units(2) units(2.5);
```

Comment:
now that's a complicated little fellow!

Reply:
Yep, the original was pretty complex too! What suggestions do you have to improve it? Should I explicitly list out each padding value (left, right, etc)